run.sh

```diff
@@ -60,16 +60,6 @@ done
 
 ./scripts/common_startup.sh $common_startup_args || exit 1
 
-# If there is a .venv/ directory, assume it contains a virtualenv that we
-# should run this instance in.
-GALAXY_VIRTUAL_ENV="${GALAXY_VIRTUAL_ENV:-.venv}"
-if [ -d "$GALAXY_VIRTUAL_ENV" -a -z "$skip_venv" ];
-then
-    [ -n "$PYTHONPATH" ] && { echo 'Unsetting $PYTHONPATH'; unset PYTHONPATH; }
-    echo "Activating virtualenv at $GALAXY_VIRTUAL_ENV"
-    . "$GALAXY_VIRTUAL_ENV/bin/activate"
-fi
```

Comment:
(just revisiting why we don't source -- I remember there being a discussion but it seems counterintuitive to me right now)